docs/root/start/sandboxes/cache.rst

```diff
@@ -38,11 +38,15 @@ A simple way to achieve this is via the `Docker Desktop <https://www.docker.com/
 
 If you have not cloned the Envoy repo, clone it with:
 
-``git clone git@github.com:envoyproxy/envoy``
+.. tabs::
 
-or
+   .. code-tab:: console HTTPS
```

Comment:
im thinking we probably want `ssh` first on the basis that most people following these instructions are devs and have a github acc

Reply:
I just did it that way because of GitHub. But it makes sense for us to have them the opposite way. I will change that.